fix(docs): 3/5 grammar in `book/` (#2215)
tact-lang/tact: docs/src/content/docs/book/expressions.mdx

Literals represent values in Tact. These are fixed values—not variables—that you _literally_ provide in your code. All literals in Tact are expressions themselves.

> You can also call [extension functions](/book/functions#extension-function) defined on certain [primitive types][p] corresponding to literals directly on the literal values:

Reviewer: ```suggestion
You can also call [extension functions](/book/functions#extension-function) defined on certain [primitive types][p] directly on their corresponding literal values:
```
